### DISCUSSION TITLE
Abema Anime

### DISCUSSION THREAD

**Discussion by luongz** [iptv-org/community#236] [Show and tell]
**Channel Name:** Abema Anime (Geoblock , using JP VPN)
**Country:** Japan
**Language:** Japanese
**Website:** https://abema.tv
**Stream URL (optional):** https://abema.tv/now-on-air/abema-anime
**Notes (optional):** There are more channel in this , hope someone can đupe this

**Comment by Dum4G** [comment]
They have protected mpd streams AFAIK

**Comment by BellezaEmporium** [comment]
Geolocked to Japan

Token retrieval : https://api.abema.io/v1/media/token?osName=pc&osVersion=1.0.0&osLang=fr&osTimezone=Europe%2FParis&appVersion=v21.818.0 (ex. for me) (forgot authentication header 'cause i'm dumb)

Link =
https://ds-linear-abematv.akamaized.net/channel/abema-anime/manifest.mpd?t= *key* &enc=wv&dt=pc_firefox&ccf=11&ut=1
